### DISCUSSION TITLE
Snowflake connectivity error

### DISCUSSION THREAD

**Discussion by mayanklalwani** [apache/superset#20855] [Q&A / Help]
Can someone help me with connecting Snowflake with Superset?

I have installed the necessary driver but while connecting, I get below erro :

"ERROR: (builtins.NoneType) None
(Background on this error at: https://sqlalche.me/e/14/dbapi)"

I have tested the username,password and account name which I am giving in the SQLAlchemy URI via a custom python script and the python script is able to successfully connect using them.

**Comment by radaniba** [comment]
@mayanklalwani did you solve this issue ? I am facing similar one
